src/server.ts

```diff
@@ -276,18 +276,25 @@ async function announce(
 		res.end(e.message);
 	}
 }
-async function statusCheck(
+
+async function status(
 	req: IncomingMessage,
 	res: ServerResponse,
 ): Promise<void> {
 	res.writeHead(200);
 	res.end("OK");
 }
+
+async function ping(req: IncomingMessage, res: ServerResponse): Promise<void> {
+	res.writeHead(200);
+	res.end("OK");
+}
+
 async function handleRequest(
```

Comment:
Since an endpoint without auth is being added, this function should refactored:
```ts
async function handleRequest(
	req: IncomingMessage,
	res: ServerResponse,
): Promise<void> {
	const allowedMethods = ["POST", "GET"];
	if (!allowedMethods.includes(req.method!)) {
		res.writeHead(405);
		res.end(
			`Methods allowed: ${formatAsList(allowedMethods, { sort: true })}`,
		);
		return;
	}

	const endpoint = req.url!.split("?")[0];
	switch (endpoint) {
		case "/api/announce":
			if (!(await authorize(req, res))) return;
			return announce(req, res);
		case "/api/webhook":
			if (!(await authorize(req, res))) return;
			return search(req, res);
		case "/api/ping":
			return ping(req, res);
		case "/api/status":
			if (!(await authorize(req, res))) return;
			return status(req, res);
		default: {
			const message = `Unknown endpoint: ${endpoint}`;
			logger.error({ label: Label.SERVER, message });
			res.writeHead(404);
			res.end(message);
			return;
		}
	}
}
```